### ISSUE TITLE
Depth lines don't line up across blank lines

### ISSUE THREAD

**Issue by frankshearar** [fsprojects/VisualFSharpPowerTools#961]
Suppose I have a function in which there's a blank line. In this event, the vertical depth lines don't line up with the non-blank lines above and below.

![off-by-one-depth-lines](https://cloud.githubusercontent.com/assets/263056/7365789/34e198c4-ed8b-11e4-9bd6-c3bfdcbb316c.png)

**Comment by dungpa** [comment]
Thanks for the bug report. 

I don't know what's causing this. Perhaps it's a WPF-related bug. The code that renders indent lines is at https://github.com/fsprojects/VisualFSharpPowerTools/blob/38715049c6c3a5b03478dd19bac47ac2c94a6382/src/FSharpVSPowerTools.Logic/DepthAdornmentManager.fs#L120-L162.

Mark this up-for-grabs. It would be great if someone can investigate further.

**Comment by yever** [comment]
I can't reproduce the non-alignment bug on my machine, but I do notice that the vertical lines are sometimes cut across (horizontally). This occurs also in @frankshearar's screenshot (you have to really zoom in to see it), between the first two lines, then after the "async" line and finally, after the second blank line. These cuts occur randomly, and seemingly regardless of blank lines.

Here's another example, with nonsense code and a lot of zooming to make it more conspicuous (though the cuts are still pretty small):
![image](https://cloud.githubusercontent.com/assets/10565807/9981699/a0b9cc0a-5fc4-11e5-9d24-bfa0a391d0bf.png)

I'd be happy to try to work on this bug, but for the time being I can only reproduce the other problem that wasn't explicitly mentioned here.

**Comment by dungpa** [comment]
@yever I no longer reproduce the original bug either. Feel free to send a pull request fixing the horizontal cuts there.

**Comment by yever** [comment]
I found it!

[This line](https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/src/FSharpVSPowerTools.Logic/DepthAdornmentManager.fs#L44) defines an "adornmentHeightFudgeFactor" which is set to 0.0, but as the comment above it goes:
```F#
// Amount to increase the adornment height to ensure there aren't gaps between adornments
// due to the way that layout rounding changes the placement of these adornments.
// Can see the bug if you set this to zero and scale the editor window to e.g. 91%, though for now I don't care
```
Well, increasing it does the trick. I played around with it, and 0.2 seems to be the smallest number that makes the gap disappear even in 400% zoom (on my screen that has 4K resolution).

I'll go make a PR.
